packages/twenty-server/src/engine/core-modules/file/services/file.service.ts

```diff
@@ -123,4 +123,28 @@ export class FileService {
 
     return [toWorkspaceFolderPath, subFolder, toFilename];
   }
+
+  extractRelativePath(fullPath: string): string {
+    const urlWithTokenMatch = fullPath.match(
+      /\/files\/attachment\/[^/]+\/([^/]+)$/,
+    );
+
+    if (urlWithTokenMatch) {
+      return `attachment/${urlWithTokenMatch[1]}`;
+    }
+
```

Comment:
Too many hardcoded things here like attachment and file folder name. It makes it hard to read and easily breakable in the future